client/app/lib/components/redux-form/Toggle.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+import MaterialToggle from 'material-ui/Toggle';
+import createComponent from './createComponent';
+import mapError from './mapError';
+
+const errorStyle = {
+  color: 'red',
```

Comment:
For consistency, maybe we can import and use red500 from http://www.material-ui.com/#/customization/colors?